### DISCUSSION TITLE
Stupid dBeaver/Pg Question

### DISCUSSION THREAD

**Discussion by opergayal49** [dbeaver/dbeaver#10779] [Q&A] [answered]
I'm a big fan of using one database gui tool for tons of different databases. And, for that, I love sequelPro on mac, but, on linux, the only similar tool I've found is dBeaver.

And, my issue: I generally use a GUI to quickly glance at table data, or, perhaps, the table layout (schema). And, on dBeaver with Postgres, that takes a LOT of clicks: Connection -> Database -> Schema -> Public -> Tables -> Table, then you can select data or properties . .

Is there an easier way, or should I try out a different tool (suggestions welcome)?

Thanks in advance! https://getappvalley.com/

**Comment by serge-rider** [comment, marked as answer]
You can switch to the Simple mode: 
![image](https://user-images.githubusercontent.com/6398845/103081290-b8cb6a00-45e8-11eb-91c7-1e54068c1267.png)

In this mode dbeaver shows just schemas and tables with no extra folders.
